### ISSUE TITLE
Colors and Smells study

### ISSUE THREAD

**Issue by LinguList** [clld/concepticon-data#434]
Not clear yet, how many concepts, and what efforts it costs, but this paper is based on a questionnaire:

* http://www.cell.com/pb/assets/raw/journals/research/current-biology/S200backup/majid.pdf

Here's a description by the MPI people:

* http://www.mpi.nl/news/hunter-gatherers-have-a-special-way-with-smells

they use charts for the colors, so naming is difficult, but the odors are based on objects, so they can be linked and will add additional information that certain things are "smelly" in our concepticon:

> For the odor task, we used ‘‘Sniffin’ Sticks’’ [61]. These are marker pens containing an odorant (instead of ink) which the participant
> can smell by removing the marker cap and smelling the tip. Sixteen odors were used: orange, leather, cinnamon, peppermint,
> banana, lemon, licorice, turpentine, garlic, coffee, apple, clove, pineapple, rose, anise, and fish. For the color task, participants
> saw 80 Munsell color chips, sampling 20 equally spaced hues at 4 degrees of brightness [60]. Odors and colors were presented
> in a randomized fixed order. After the free naming task, participants were shown a sheet with all 80 color chips laid out according
> to hue and brightness, plus 4 additional achromatic colors, and asked to identify the best example of the most frequent colors elicited
> from the naming task [62].

**Comment by LinguList** [comment]
Supplementary data is in the Language Archive:

* https://corpora.humlab.lu.se/ds/asv/;jsessionid=B78FEC6C55780E02B2B5E91962C9E874?0&openhandle=hdl:10050/00-0000-0000-0003-CFA7-6

**Comment by LinguList** [comment]
In the SM, they mention some kind of a Munsell code for colors. We may consider adding this as metadata to concepticon? But we'd need a resource where one can find a chart, and then link the colors in concepticon to those codes (making some rough, bold decisions, no need to be too strict).

**Comment by SimonGreenhill** [comment]
Yes, it would be awesome to link colors to a real Munsell-style catalogue. This would allow us to look into things like [Berlin and Kay's](https://en.wikipedia.org/wiki/Basic_Color_Terms:_Their_Universality_and_Evolution) work
